CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.3] - 2019-06-28
```

Comment:
0.1.4? Never mind, but ideally get someone to do a quick review on PRs.

Reply:
Yeah, I've already fixed it in the next commit.